src/core/analytics/analyticsreporter.js

```diff
@@ -109,11 +109,13 @@ export default class AnalyticsReporter {
       ytag('optin', true);
       cookieData = ytag('yfpc', null);
     } else if (this._conversionTrackingEnabled) {
-      throw new AnswersAnalyticsError('Tried to enable conversion tracking without including ytag');
```

Comment:
nit: the import for this can be removed now. looks like we don't have the GH check for linting